Source/Core/Common/BitField.h

```diff
@@ -183,7 +183,7 @@ struct BitField
 
 	__forceinline StorageType GetMask() const
 	{
-		return ((~(StorageTypeU)0) >> (8 * sizeof(T) - bits)) << position;
+		return (((StorageTypeU)~0) >> (8 * sizeof(T) - bits)) << position;
```

Comment:
Seems to me that this would break if StorageType is 64-bit, as ~0 is a 32-bit value.